app/Promotions/MedusaPromotions.php

```diff
@@ -114,6 +114,7 @@ private function getBranchForReq()
     public function getPromotableInfo($payGrade2Check = null, $sfcCheck = true, $path = null)
     {
         $flags = [
```

Comment:
@cinnion Let's go ahead and rename this variable, since it's now not just flags

Reply:
Renamed to $info, short for PromotableInfo.